app/models/group/unique_nextcloud_group.rb

```diff
@@ -0,0 +1,21 @@
+# encoding: utf-8
+
+#  Copyright (c) 2012-2023, Jungwacht Blauring Schweiz. This file is part of
+#  hitobito and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito.
+
+module Group::UniqueNextcloudGroup
+  extend ActiveSupport::Concern
+    
+  included do
+    self.nextcloud_group = :calculate_nextcloud_group
+  end
+
+  def calculate_nextcloud_group
+    {
+      "gid" => "#{group_id.to_s}_#{self.type}",
```

Comment:
```suggestion
      "gid" => "#{group_id}_#{type}",
```

Das `.to_s` wird implizit von der Stringinterpolation ausgeführt und muss daher nicht geschrieben werden. `self.` braucht es eigentlich nicht, wenn man lediglich Attribute auslesen möchte. Da ich das hier nur im Webinterface sehe, kann ich natürlich einen Grenzfall übersehen, bei dem das doch nötig ist. Gewissheit bringen hier sicherlich entsprechende Specs